app/templates/components/preprint-form-header.hbs

```diff
@@ -26,7 +26,7 @@
             <em class="m-r-md"> {{t "components.preprint-form-header.server" documentType=provider.documentType}}: </em>
             {{#if currentProvider}}
                 <img alt="provider logo" class="current-provider" title="{{currentProvider.name}}" src={{provider-asset currentProvider.id 'square_color_no_transparent.png'}}>
-                {{currentProvider.name}}
+                {{if isOSFPreprints 'OSF Preprints' currentProvider.name}}
```

Comment:
Where is `isOSFPreprints` coming from? I don't see this var anywhere?

Reply:
It's defined on submit controller below and is passed down as argument to any of the child components that could use the special casing. Alternatively, it could be also computed on the template or be added to the preprint provider model, but I thought it's cleaner to compute once and pass it down as arguments to other components.

https://github.com/CenterForOpenScience/ember-osf-preprints/blob/874030da78b0ddc9bcdcb31d034061b901ba53e2/app/controllers/submit.js#L289-L291